maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/ConsoleOutputFileReporter.java

```diff
@@ -69,8 +69,8 @@ public void close()
         {
             try
             {
-                fileOutputStream.flush();
-                fileOutputStream.close();
+                // fileOutputStream.flush(); Will not call close on exception!
+                fileOutputStream.close(); // Will implicitly flush.
```

Comment:
Writer needs close() without flush() but FileOutputStream does not have implicit flush while closing. Javadoc does not mention it. Implementation in Oracle JDK is maybe different and probably flushes on close but that's this JVM vendor.